src/go/rpk/pkg/cli/cmd/api.go

```diff
@@ -27,7 +27,7 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-func NewApiCommand(fs afero.Fs) *cobra.Command {
+func NewApiCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
```

Comment:
Since `Manager` encapsulates `afero.Fs`, should it be abstracted out and make everything use the manager instead? There is a mixed pattern in the code.

It seems like the right way, but I'm not sure :thinking:

Reply:
I initially had the same thought, but although the manager needs access to the filesystem, so do other commands, and for different reasons.

Reply:
Exactly :D